### ISSUE TITLE
 Error at mounting ssl certificate to frontend container

### ISSUE THREAD

**Issue by DoohyunHwang97** [swsnu/swppfall2022#212]
<img width="572" alt="스크린샷 2022-12-09 오전 3 36 15" src="https://user-images.githubusercontent.com/76721027/206539382-3d9010ac-5ec9-46fd-a6cd-79fa6c33e038.png">
<img width="763" alt="image" src="https://user-images.githubusercontent.com/76721027/206540078-cc4caabb-fc30-4624-88e1-5e022592dc9e.png">
<img width="425" alt="image" src="https://user-images.githubusercontent.com/76721027/206540198-693905e2-0d2d-4774-b4ff-f62c1c3d6634.png">

ssl certificate을 컨테이너에 마운팅하는 과정에서 컨테이너를 런했을 때 nginx 에러가 뜨는데
해당에러가 뜨는 이유를 잘 모르겠어서 도움 받고 싶습니다!
감사합니다

**Comment by gajagajago** [comment]
`chmod`를 하셨나요?

**Comment by DoohyunHwang97** [comment]
sudo chmod -R 755 /etc/letsencrypt/archive/stylestargram.shop/fullchain1.pem 이런식으로 했는데
혹시 sudo chmod -R 755 뒤에 붙이는 path에 뭐를 붙이면 될지 알 수 있을까요?!

**Comment by diffzerg** [comment]
no such file 뜨는것 보니, 제가 올렸던 이슈 한번 확인해보세요 (run_docker_frontend에서 경로 뒷부분 / 제거)

**Comment by DoohyunHwang97** [comment]
감사합니다! 알려주신데로 했는데 그대로네요ㅜ
run_docker_frontend.sh 파일은 따로 돌려줘야되는건지 아니면 도커런할 때 알아서 돌아가는건지 잘 모르겠어서 혹시 아시나요?

**Comment by gajagajago** [comment]
@DoohyunHwang97 
- You may have to chmod -R 755 ssl certificate **directory**: 실제 fullchain과 privkey가 있는 '디렉토리'를 chmod -R 해 보시면 좋을 것 같습니다. 
- `run_docker_frontend.sh` 는 직접 실행해주셔야 합니다.

**Comment by DoohyunHwang97** [comment]
> * chmod -R 755

<img width="569" alt="image" src="https://user-images.githubusercontent.com/76721027/206654055-70a7ae9a-74fa-4ea4-8185-f66501e6d122.png">
이렇게 하는게 맞을까요?

**Comment by gajagajago** [comment]
네 컨테이너 안에 들어가셔서 full chain이랑 privkey가 COPY가 제대로 되었는지 확인 가능할까요?

**Comment by DoohyunHwang97** [comment]
<img width="569" alt="스크린샷 2022-12-09 오후 7 34 01" src="https://user-images.githubusercontent.com/76721027/206684101-54efd1df-915e-4797-a3b4-ed0542077b38.png">
<img width="564" alt="스크린샷 2022-12-09 오후 7 39 05" src="https://user-images.githubusercontent.com/76721027/206684135-24d79798-a57a-47da-85d5-e1a985911c99.png">
컨테이너 안에서 full chain이랑 privkey가 COPY 확인했습니다
오류가 떴을때는 docker run을 sudo docker run --rm \--name "frontend" \-p 3000:3000 -it \frontend:latest으로 바로 실행했었는데
sh run_docker_frontend.sh을 실행하니 내부의 docker run이 실행되면서 오류가 뜨지 않는 것 같습니다
하지만 컨테이너에도 들어가고 작동을 확인했는데 docker container ls을 해도 frontend container가 뜨지 않는 것을 보아 실행되지 안 되는 것 같은데 이유를 모르겠습니다ㅠ
<img width="572" alt="image" src="https://user-images.githubusercontent.com/76721027/206686532-4c18619c-01aa-4535-a7ce-888446a81728.png">
감사합니다

**Comment by gajagajago** [comment]
도커 컨테이너 내에서 sh run_docker_frontend.sh 를 하셨다는 것인가요?

**Comment by DoohyunHwang97** [comment]
<img width="565" alt="image" src="https://user-images.githubusercontent.com/76721027/206701494-a0a34b17-e51a-4751-9965-334a991d3200.png">
![KakaoTalk_Image_2022-12-09-21-22-58](https://user-images.githubusercontent.com/76721027/206702114-258a5172-6e85-40ec-89a0-3d7d97e61a6e.jpeg)

컨테이너가 돌아가고 있는데 역시 접속은 안됩니다ㅠ
내부에 nginx 실행시켜봐도 안되네요 nginx 내부에서 문제가 생긴걸까요?

**Comment by DoohyunHwang97** [comment]
@gajagajago 
넵 감사합니다!!ㅠㅠ
내일 연구실에 언제 계시는지 알수 있을까요??

**Comment by gajagajago** [comment]
10 - 6 있습니다 ㅎㅎ 방문시간 메일 주시면 해당 시간에 자리에 있겠습니다

**Comment by DoohyunHwang97** [comment]
넵 감사합니다!! 연락드리겠습니다:)

**Comment by gajagajago** [comment]
해결되었습니다.
